docs/modules/admin/pages/installing.adoc

```diff
@@ -1,37 +1,63 @@
-= Installing
-
-Trustification services are all shipped using the `trust` CLI which is capable of running any of the services using command line flags.
-
-Trustification rely on external services for storage and eventing.  Whether you want to run on-prem or in the cloud, there is an alternative for both.
-
-To store data, trustification services rely on the Amazon S3 APIs. This does not mean you need to run trustification on AWS, because the API is implemented by Google Cloud Storage (GCS) and MinIO (Open Source) as well.
-
-For notification and sharing events between trustification services, a message bus like Apache Kafka or AWS SQS can be used.
-
-== Infrastructure
-
-At present, the following infrastructure must exist (names can be changed by adjusting command line flags in manifests):
-
-* S3 buckets named `vexination` and `bombastic` for storing data and the search index.
-* Topics/queues named `sbom-stored`, `sbom-indexed` and `sbom-failed` for Bombastic.
-* Topics/queues named `vex-stored`, `vex-indexed` and `vex-failed` for Vexination.
-* Credentials for reading and writing data to the above resources.
-
-Depending on your choice of infrastructure, you can create these resources and manage access using your favorite tool, such as terraform.
-
-== Deployment options
-
-=== Bare metal
-
-Running on bare metal requires downloading the `trust` binary from the link:https://github.com/trustification/trustification/releases[releases]. 
```

Comment:
I followed the link here to "releases", but didn't find the "trust" binary file to download.  I did find the zip & tar.gz source code to compile the "trust" binary.  Is that what was meant by saying "downloading" here?

I moved this sentence into the "index.adoc" file, and changed it slightly, see line 20 of "index.adoc" for the rewritten sentence.

Reply:
The binary actually has the suffix of the target OS and CPU architecture, so it is named trust-linux-amd64. It's a bit hard to find though, so probably worth expanding on in the doc.

Reply:
Okay, thanks for the additional information.  I'll expand the docs.  Also, I'll do some testing, and I'll reach out to the team if I have questions.